packages/client/components/PokerActiveVoting.tsx

```diff
@@ -100,8 +100,11 @@ const PokerActiveVoting = (props: Props) => {
   const {id: stageId, scores} = stage
   const hasVotes = scores.length > 0
   const isFacilitator = viewerId === facilitatorUserId
-  const viewerHasVoted = Boolean(scores.find(({userId}) => userId === viewerId))
-  const checkedInCount = meetingMembers.length
+  const viewerHasVoted = useMemo(() => !!scores.find(({userId}) => userId === viewerId), [scores])
+  const checkedInCount = useMemo(
+    () => meetingMembers.filter(({isSpectating}) => !isSpectating).length,
```

Comment:
I think it is better to do this on server side as well, and publish the scores on updating `isSpectating`, see my test results

Reply:
Nice catch with the test! That was happening because I was removing the viewer's vote in the stage where they clicked "I don't vote". However, it looks like "I don't vote" applies to all stages, so I'm now removing their vote in all stages. I've moved this logic to the server. 

I think the `checkedInCount` needs to be calculated on the client as we want to know the total number of meetingMembers that are not spectating. Update pushed!